bin/upload_repo_rpm

```diff
@@ -34,8 +34,11 @@ for f in build/*."el$DISTRO".{x86_64,aarch64}.rpm; do
         DESTINATION="dev"
       fi
     fi
-
-    ./bin/sign_rpm "$GPG_SECRET" "$f"
+   
+    
+    cp $f /work/repomanager/docker/rpm_signer/data/$DISTRO/$ARCH/
+    /usr/bin/docker exec rpm_signer /worker.sh $DISTRO $ARCH
+    mv /work/repomanager/docker/rpm_signer/data/$DISTRO/$ARCH/*.rpm build/
```

Comment:
Why do we need to move the file to sign and move it back? 
Not sure that docker in that way will work in Github actions, but I need to double-check.
Otherwise – what about the solution to run like `docker run ... -v ... -rm rpm_signer` with binding volume from the host machine?